web-wallet/src/lib/components/ContractStatusesList/ContractStatusesList.svelte

```diff
@@ -34,7 +37,15 @@
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
-    gap: 0.625em;
+    gap: 0.5em;
```

Comment:
Maybe let's use the `--small-gap` var from the design language?

Reply:
As mentioned above, the Figma needs to be leading for specification as the tokens are going to be programatically pulled from Figma's API in Duskit.  Therefore we need to figure this out at the Figma level as the gap value is `0.5em` there.

Reply:
No, no. 0.5 is fine. But the small gap variable is also 0.5em, so I think it's better to reuse the variable from the design language.